tailwind.config.ts

```diff
@@ -13,8 +13,26 @@ const config: Config = {
         "gradient-conic":
           "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
       },
+      fontFamily: {
+        sans: ["Inter", "sans-serif"],
+        serif: ["Georgia", "serif"],
+      },
+      darkMode: {
+        primary: "#1a202c",
+        secondary: "#2d3748",
+        accent: "#3b82f6",
+        background: "#0e172a",
+        text: "#ffffff",
+      },
+      lightMode:{
+        primary:"#395B64",
+        background:"#FFFFFF",
+        text:"#000000"
+        
+      },
```

Comment:
no this is not what I mean I mean themes like if it;s blue, light blue, text colors